### PR TITLE
Add a xonsh script which reads the circleci config and generates reference images

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

**Comment by bsipocz** [comment]
Could you rebase please?

File: gh_issuereport.py [hunk 1]
```diff
@@ -125,20 +125,12 @@ def count_prs_since(dt, repo, auth=None, verbose=True, cacheto=None):
...
 
-    url = 'http://pypi.python.org/pypi/{0}/{1}'.format(pkg, version)
+    datestr = j['releases'][version][0]['upload_time']
 
-    dom = minidom.parseString(requests.get(url).content)
-
-    table = dom.getElementsByTagName('table')[0]
-    t = table.getElementsByTagName('tr')[1].getElementsByTagName('td')[-2].firstChild
-    if t.nodeType != t.TEXT_NODE:
-        raise ValueError("pypi page for {0}/{1} ddoesn't seem to have a date".format(pkg, version))
```
Comment (bsipocz) [inline gh_issuereport.py]: I suppose in the json there is always a date?

Comment (Cadair) [reply gh_issuereport.py]: there is in everyone I have seen, but small sample size.

**Comment by bsipocz** [comment]
Please ping me when you're ready with this.

**Comment by Cadair** [comment]
@bsipocz I think this is good

**Comment by pllim** [comment]
We have since revamped the whole thing. This is no longer needed.

**Pull request closed without merging**